open-metadata-implementation/admin-services/admin-services-configuration.postman_collection.json

```diff
@@ -202,12 +202,12 @@
 					"raw": ""
 				},
 				"url": {
-					"raw": "http://localhost:8080/open-metadata/admin-services/users/garygeeke/servers/cocoMDS1/server-user-id?id=cocoMDS",
-					"protocol": "http",
+					"raw": "https://localhost:9443/open-metadata/admin-services/users/garygeeke/servers/cocoMDS1/server-user-id?id=cocoMDS",
```

Comment:
Another key question - are we ok with 9443 as the default. I know you mentioned this in the issue and indeed referred to 8443.

I'd actually be inclined to suggest the chassis (ie core egeria) sticks with 8443 - and we move the UI to 9443.